sample-apps/grpc-greeter-go/greeter-k8s.template.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: greeter
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: greeter
+  template:
+    metadata:
+      labels:
+        app: greeter
+    spec:
+      containers:
+      - name: greeter
+        image: gcr.io/$GOOGLE_CLOUD_PROJECT/grpc-greeter-go-server
```

Comment:
Why only a deployment for the server, not the client?

Reply:
The client isn't a long-running process, it just makes some requests to the server and then exits. We could create a k8s job for it, if you think that'll be useful?

Reply:
Ok! No, I think just the one deployment is fine, then.